AUTHORS

```diff
@@ -2,5 +2,5 @@ The PyFCP applications and library were written
 by David McNab (aum) in May-June, 2006.
 
 Contact the author at:
-  david@freenet.org.nz
+  david@rebirthing.co.nz
```

Comment:
I wonder if David still wants to be listed as maintainer for this codebase. I suspect not.